### PR TITLE
chapter 14: follow user

### PR DESCRIPTION
**Pull request by binhlt-0028**
image
chapter 14: following and follower

### CODE REVIEW

**Comment by binhlt-0028** [comment]
ready

File: app/models/relationship.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
```
Comment (hungcnh-0828) [inline app/models/relationship.rb]:
```suggestion
  belongs_to :follower, class_name: User.name
```

File: app/models/user.rb [hunk 2]
```diff
@@ -1,5 +1,11 @@
 class User < ApplicationRecord
   has_many :microposts, dependent: :destroy
+  has_many :active_relationships, class_name: "Relationship",
```
Comment (hungcnh-0828) [inline app/models/user.rb]:
```suggestion
  has_many :active_relationships, class_name: Relationship.name,
```

File: app/models/user.rb [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+            foreign_key: "follower_id", dependent: :destroy
```
Comment (hungcnh-0828) [inline app/models/user.rb]:
```suggestion
            foreign_key: :follower_id, dependent: :destroy
```

**Pull request merged**